### PR TITLE
DOI badge

### PR DESCRIPTION
**Pull request by dpo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,4 +42,4 @@ pkg> test ExpressionTreeForge
 See the [tutorial](https://JuliaSmoothOptimizers.github.io/ExpressionTreeForge.jl/dev/tutorial/).
 
 ## Dependencies
-This module is used together with [PartitionedStructures.jl](https://github.com/JuliaSmoothOptimizers/PartitionedStructures.jl) by [PartiallySeparableNLPModels.jl](https://github.com/paraynaud/PartiallySeparableNLPModels.jl) and [PartiallySeparableSolvers.jl](https://github.com/paraynaud/PartiallySeparableSolvers.jl) to define a trust-region method exploiting partial separability through partitioned quasi-Newton approximations. 
\ No newline at end of file
+This module is used together with [PartitionedStructures.jl](https://github.com/JuliaSmoothOptimizers/PartitionedStructures.jl) by [PartiallySeparableNLPModels.jl](https://github.com/paraynaud/PartiallySeparableNLPModels.jl) and [PartiallySeparableSolvers.jl](https://github.com/paraynaud/PartiallySeparableSolvers.jl) to define a trust-region method exploiting partial separability through partitioned quasi-Newton approximations.
```
Comment (paraynaud) [inline README.md:45]:
```suggestion
This module is used together with [PartitionedStructures.jl](https://github.com/JuliaSmoothOptimizers/PartitionedStructures.jl) by [PartiallySeparableNLPModels.jl](https://github.com/JuliaSmoothOptimizers/PartiallySeparableNLPModels.jl) and [PartiallySeparableSolvers.jl](https://github.com/JuliaSmoothOptimizers/PartiallySeparableSolvers.jl) to define a trust-region method exploiting partial separability through partitioned quasi-Newton approximations.
```

**Review by paraynaud — COMMENTED** [review]
We can change `paraynaud` to `JuliaSmoothOptimizers`.

File: README.md (hunk 1 above)
Comment (paraynaud) [reply README.md:45]: I don't know if this change is irrelevant in this PR.

Comment (dpo) [reply README.md]: Yes, thank you.

**Pull request merged**